src/JoomlaBrowser.php

```diff
@@ -656,12 +656,11 @@ public function selectMultipleOptionsInChosen($label, $options)
 	 */
 	public function doAdministratorLogout()
 	{
-		$this->click(['xpath' => "//ul[@class='nav nav-user pull-right']//li//a[@class='dropdown-toggle']"]);
+		$this->click(['css' => ".nav-link > .fa-user"]);
```

Comment:
you could actually move them to the locator class, making it reusable, https://github.com/joomla-projects/joomla-browser/blob/v4.0.0/src/Locators/Locators.php